apimonitor/monitor/serializers.py

```
@@ -0,0 +1,81 @@
+class ServiceSerializer(serializers.ModelSerializer):
+    """Serializer for Service model."""
+    
+    full_url = serializers.ReadOnlyField()
+    
+    class Meta:
+        model = Service
+        fields = [
+            'id', 'name', 'base_url', 'health_path', 'full_url',
+            'method', 'headers', 'timeout_s', 'enabled',
+            'created_at', 'updated_at'
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)